kernel/kernel-config-4.14.x

```diff
@@ -2209,6 +2218,7 @@ CONFIG_TMPFS_POSIX_ACL=y
 CONFIG_TMPFS_XATTR=y
 CONFIG_HUGETLBFS=y
 CONFIG_HUGETLB_PAGE=y
+CONFIG_ARCH_HAS_GIGANTIC_PAGE=y
```

Comment:
It's not clear to me if all of these are related to enabling the hotplug memory or not (as noted by @jodh-intel ). @devimc , can you tell us if some of these are just kernel version bump related or not?
Also, have we stared hard at all the items that got turned on, and ensured that we really do need/want them turned on for our container kernel. Previously we've had things get turned on as the default, but they have not been needed by us. Thus, I'd like to ensure each newly enabled item has been checked for validity (or we may slowly bloom the kernel with things we don't actually need).

Yes, kernel config is complex and inter-twined. There have been efforts to make it easier and more understandable, but we are still with what we have - it does the job, and fixing it is a big hard problem...